docs/install/docker-plugin.rst

```diff
@@ -4,6 +4,8 @@
 Installing the Flocker Plugin for Docker
 ========================================
 
+:ref:`docker-plugin` allows Flocker to manage your data volumes while using other tools such as Docker, Docker Swarm, or Mesos to manage your containers.
```

Comment:
Seems a shame that we just have to copy that sentence from the linked page, but fine as is. I'll be merging this now.